packages/lib/gpcircuits/scripts/parameters/paramGen.ts

```diff
@@ -0,0 +1,553 @@
+// This file is the source of truth for GPC production circuit configurations.
+//
+// Run `yarn gen-circuit-parameters && yarn gen-artifacts` after editing.
+import { ProtoPODGPCCircuitParams } from "../../src";
+
+// Circuit parameters are grouped by numbers of objects and arranged in
+// ascending order wrt. number of entries.
+export const PARAMS: ProtoPODGPCCircuitParams[] = [
+  /*
+   * 1 object
+   */
+  {
```

Comment:
I suggest some even smaller options for very basic use cases: 1o1e and 1o5e with ov4

Reply:
Looking at the cost, I think this is particularly important, because the minimum size has gone up from 6K to 18K.  I think we should have 1o1e and 1o5e circuits with no other modules, and then with numericic+inequality+tuple+list modules.